src/config.c

```diff
@@ -3126,6 +3127,7 @@ standardConfig static_configs[] = {
     /* Integer configs */
     createIntConfig("databases", NULL, IMMUTABLE_CONFIG, 1, INT_MAX, server.dbnum, 16, INTEGER_CONFIG, NULL, NULL),
     createIntConfig("port", NULL, MODIFIABLE_CONFIG, 0, 65535, server.port, 6379, INTEGER_CONFIG, NULL, updatePort), /* TCP port. */
+    createIntConfig("bjm-threads", NULL, IMMUTABLE_CONFIG, 1, 128, server.bjm_threads_num, 2, INTEGER_CONFIG, NULL, NULL),
```

Comment:
I think this practically needs to be 1 as a default, otherwise users might see significantly more usage than before.